tests/common.py

```diff
@@ -14,7 +14,7 @@
 
 TEST_CONN = os.getenv('SQLA_TEST_CONN', 'postgresql:///cfdm_unit_test')
 
-rest.app.config['NPLUSONE_RAISE'] = True
+rest.app.config['NPLUSONE_RAISE'] = False
```

Comment:
Yea, Josh helped me with that before. 

Also we are not getting nplusone errors on develop, where we are checking for it. 
https://github.com/18F/openFEC/blob/develop/tests/common.py#L17